packages/core/plugins/core-plugins-browser-mocks/src/plugins_service.mock.ts

```diff
@@ -27,7 +28,10 @@ const createStartContractMock = () => {
   return startContract as PluginsServiceSetup;
 };
 
-const createPluginInitializerContextMock = (config: unknown = {}) => {
+const createPluginInitializerContextMock = (
+  config: unknown = {},
+  { buildFlavor = 'serverless' }: { buildFlavor?: BuildFlavor } = {}
```

Comment:
For UI-only changes, using the config should be ok although core now has an [elasticsearch capabilities](https://github.com/elastic/kibana/pull/164850) API that gets info from the cluster directly. 

The new API is preferable over assuming an always stateless elasticsearch because KIbana's sometimes run against a stateful ES instance.  If that's not a problem here, then all's good.

Reply:
Yea, this is fine.